openfisca_core/periods.py

```diff
@@ -358,6 +358,41 @@ def __str__(self):
             if fragment is not None
             )
 
+    def __cmp__(self, other):
```

Comment:
You may want to define the `compare_period_size(x,y)` function is the `periods` module though, for better encapsulation.